src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -223,6 +224,22 @@ private void CheckFileSystem()
             }
         }
 
+        private void ValidateLinuxSKUConfiguration()
+        {
+            var url = _environment.GetEnvironmentVariable(ScmRunFromPackage);
+
+            if (string.IsNullOrEmpty(url) && _environment.IsLinuxConsumptionOnAtlas() && _environment.IsManagedAppEnvironment())
+            {
+                _logger.LogWarning($"Zip deployment is not supported on a Linux Consumption app configured to use Managed Identity. Functions may not be indexed correctly.");
```

Comment:
We need to perform the check to validate MI is being used.

Reply:
Added a similar check as used here - https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.WebHost/Health/TokenExpirationService.cs#L62-#L65. Is this sufficient for MI check?

```csharp
private void AnalyzeSasTokenInUri()
        {
            // If AzureWebJobsStorage__accountName is set, we are using identities and don't need to check for SAS token expiration
            if (!string.IsNullOrEmpty(_environment.GetEnvironmentVariable($"{EnvironmentSettingNames.AzureWebJobsSecretStorage}__accountName")))
            {
```